server/routes/creatingLicences/index.ts

```diff
@@ -69,22 +69,23 @@ export default function Index({
       asyncMiddleware(handler),
     )
 
-  const post = (path: string, handler: RequestHandler, type?: new () => object) =>
+  const postWithHardStopCheck = (path: string, handler: RequestHandler, type?: new () => object) =>
     router.post(
       routePrefix(path),
       roleCheckMiddleware(['ROLE_LICENCE_RO']),
       fetchLicence(licenceService),
       validationMiddleware(conditionService, type),
+      hardStopCheckMiddleware(UserType.PROBATION),
       asyncMiddleware(handler),
     )
 
-  const postWithHardStopCheck = (path: string, handler: RequestHandler, type?: new () => object) =>
+  const postWithPreLicenceCreationCheck = (path: string, handler: RequestHandler, type?: new () => object) =>
```

Comment:
This was a follow on from the previous PR to add pre licence creation to the POST request.